tests/integration/ex21_extraction_test.py

```diff
@@ -0,0 +1,37 @@
+"""Integration testing for labeling entities with LayoutLM and extracting Ex. 21 tables."""
+
+import pandas as pd
+from mozilla_sec_eia.ex_21.inference import create_inference_dataset, perform_inference
+from mozilla_sec_eia.ex_21.train_extractor import LABELS, load_model
+from pandas.testing import assert_frame_equal
+from transformers import AutoProcessor
+
+
+def test_model_loading():
+    """Test loading a fine-tuned LayoutLM model from MLFlow."""
+    load_model()
+
+
+def test_dataset_creation(test_dir):
+    pdf_dir = test_dir / "data/test_pdfs"
+    dataset = create_inference_dataset(pdfs_dir=pdf_dir)
+    assert dataset.shape == (2, 4)
+
+
+def test_inference_and_table_extraction(test_dir):
+    """Test performing inference and extracting an Ex. 21 table."""
+    model = load_model()
+    processor = AutoProcessor.from_pretrained(
+        "microsoft/layoutlmv3-base", apply_ocr=False
+    )
+    pdf_dir = test_dir / "data/test_pdfs"
```

Comment:
I added the test PDFs directly to a data directory in the repo tests. Not sure if this is the best way.